AndroidApplicatie/app/src/main/java/com/example/projectd/CameraActivity.java

```diff
@@ -0,0 +1,90 @@
+package com.example.projectd;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.app.ActivityCompat;
+
+import android.Manifest;
+import android.content.ContentValues;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.net.Uri;
+import android.os.Bundle;
+import android.provider.MediaStore;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.Toast;
+
+public class CameraActivity extends AppCompatActivity {
+
+    //Permission code voor toestemming camera & opslag
+    private static final int PERMISSION_CODE = 1000;
+    private static final int IMAGE_CAPTURE_CODE = 1001;
+    Button mMaakFotoBtn;
+    ImageView mImageView;
+    Uri afbeelding_uri;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_camera);
+
+        //Assign uit de layout
+        mImageView = findViewById(R.id.image_view);
+        mMaakFotoBtn = findViewById(R.id.camera_maakFoto_btn);
```

Comment:
of engelse of nederlandse variabele namen

Reply:
Goed punt